test/e2e/storage/testsuites/volumes.go

```diff
@@ -72,21 +72,18 @@ func (t *volumesTestSuite) getTestSuiteInfo() TestSuiteInfo {
 	return t.tsInfo
 }
 
-func (t *volumesTestSuite) skipUnsupportedTest(pattern testpatterns.TestPattern, driver TestDriver) {
+func (t *volumesTestSuite) skipUnsupportedTest(pattern testpatterns.TestPattern, driver TestDriver) bool {
+	return false
 }
 
-func skipPersistenceTest(driver TestDriver) {
+func skipPersistenceTest(driver TestDriver) bool {
```

Comment:
also seems like a double negative

Reply:
I've inlined the code from these functions and removed them. They were trivial and now `testVolumes` is easier to understand.